gtest_parallel.py

```diff
@@ -559,7 +559,16 @@ def main():
   if options.dump_json_test_results is not None:
     test_results = CollectTestResults(options.dump_json_test_results)
 
-  save_file = os.path.join(os.path.expanduser("~"), ".gtest-parallel-times")
+  if sys.platform == 'win32':
+    default_cache_path = os.path.join(os.path.expanduser('~'), 'AppData', 'Local')
+    cache_path = os.environ.get('LOCALAPPDATA', default_cache_path)
+  else:
+    # We don't use xdg module since it's not a standard.
+    default_cache_path = os.path.join(os.path.expanduser('~'), '.cache')
+    cache_path = os.environ.get('XDG_CACHE_HOME', default_cache_path)
+
+  save_file = os.path.join(cache_path, 'gtest-parallel')
```

Comment:
Initially I added this code but it looks like this directory has to be created with special permissions. On my machine it's 0700 but I couldn't find exact value in the XDG specification (what about sticky bit?). Also, there is other code which assumes that user's home directory always exists. I understand that there is a huge difference between home and cache but I thought that it's safer to assume it exists. :)

So, I missed that now we can set XDG_CACHE_HOME to an arbitrary value and thus we have to ensure it exists.